Bottom: 2712163046e0efb6e4e82cb93b94127aae088919
Top:    da0b296334a36dadc4101fc078c3f1d99cf3655d
Author: Ian Kent <raven@themaw.net>
Date:   2013-05-02 14:46:27 +0800

Refresh of autofs-5.0.7-use-MS_UNBINDABLE.patch

---

diff --git a/daemon/indirect.c b/daemon/indirect.c
index ad5a82b..2b0ea6e 100644
--- a/daemon/indirect.c
+++ b/daemon/indirect.c
@@ -146,7 +146,7 @@ static int do_mount_autofs_indirect(struct autofs_point *ap, const char *root)
 	if (!type || strcmp(ap->entry->maps->type, "hosts"))
 		map_name = ap->entry->maps->argv[0];
 
-	ret = mount(map_name, root, "autofs", MS_UNBINDABLE|MGC_VAL, options);
+	ret = mount(map_name, root, "autofs", MS_UNBINDABLE|MS_MGC_VAL, options);
 	if (ret) {
 		crit(ap->logopt,
 		     "failed to mount autofs path %s at %s", ap->path, root);
